docassemble/assemblylinewizard/test_map_names.py

```diff
@@ -52,6 +52,12 @@
   # "county_division": not implemented,
   "court_address_county": "courts[0].address.county",
   "court_county": "courts[0].address.county",
+  
+  # Singluar prefixes that still have suffixes
+  "trial_court": "trial_court",
+  "trial_court_address_county": "trial_court.address.county",
+  "trial_court_county": "trial_court.address.county",
+  "trial_court_division": "trial_court.division",
```

Comment:
We should also reserve trial_court.department--sorry this probably wasn't handled in the past.